crates/iota/src/validator_commands.rs

```diff
@@ -1075,7 +1061,9 @@ pub async fn get_validator_summary(
         .await?;
     let (active_validators, pending_active_validators_id) = match iota_system_state {
         IotaSystemStateSummary::V1(v1) => (v1.active_validators, v1.pending_active_validators_id),
-        IotaSystemStateSummary::V2(v2) => (v2.active_validators, v2.pending_active_validators_id),
+        IotaSystemStateSummary::V2(v2) => {
+            (v2.to_committee_members(), v2.pending_active_validators_id)
```

Comment:
But then we would miss all active validators that are not in the subset of the committee members IMO.

Reply:
Makes sense. Fixed. Probably also need to update `ValidatorStatus` to include `CommitteeMember`, it might be useful in showing validator info.

Reply:
@thibault-martinez do you think that would be something useful your team could implement in another PR?

Reply:
Created an issue https://github.com/iotaledger/iota/issues/6115